Controller/ForgotResetController.php

```diff
@@ -93,19 +95,19 @@ public function forgotPasswordAction(Request $request)
 
                     $request->getSession()->getFlashBag()->add(
                         "success",
-                        "Check your email for instructions on how to reset your password."
+                        $generic_msg //"Check your email for instructions on how to reset your password."
                     );
                     return $this->redirect($this->generateUrl('_login'));
```

Comment:
We should also be consistent about redirecting the user to avoid hinting at success.